app/controllers/arclight/repositories_controller.rb

```diff
@@ -6,5 +6,12 @@ class RepositoriesController < ApplicationController
     def index
       @repositories = Arclight::Repository.all
     end
+
+    def show
+      repository = Arclight::Repository.find_by!(slug: params[:id])
+      redirect_to search_action_path(
```

Comment:
Ok, my question is also about the url.. should the vanity slug url persist or is a redirect to a catalog faceted one ok? @anarchivist